packages/db/src/lib/granule.ts

```diff
@@ -354,3 +360,61 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
+ *
+ * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
```

Comment:
Nit: Should this be either?    I think we probably *don't* want to allow a transaction to create a subtransactio here